### PR TITLE
Update index-v7.md

### PR DESCRIPTION
**Pull request by TwoMoreThings**
Added note about ensuring you inherit from SurfaceController and added the name to use for the partial view

### CODE REVIEW

File: Getting-Started/Code/Creating-Forms/index-v7.md [hunk 1]
```diff
@@ -37,6 +37,8 @@ Build your solution after adding the model.
 ### Creating the view
 Next, we add the view for the form to the `/View/Partials` folder. Because we've added the model and built the solution we can add it as a strongly typed view.
 
+Name your view "ContactForm"
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index-v7.md]:
```suggestion
Name your view "ContactForm".
```

File: Getting-Started/Code/Creating-Forms/index-v7.md [hunk 2]
```diff
@@ -86,6 +88,8 @@ namespace MyFirstForm.Controllers
 }
 ```
 
+You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio
```
Comment (sofietoft) [inline Getting-Started/Code/Creating-Forms/index-v7.md]:
```suggestion
You should note that the controller inherits "SurfaceController" and not from "Controller" as initially added to the file by Visual Studio.
```

**Comment by TwoMoreThings** [comment]
looks good to me,
t

**Comment by sofietoft** [comment]
Awesome! 😄 
Would you mind committing the suggestions?
I don't have permissions to do that on your patch 🙈

**Comment by sofietoft** [comment]
Thanks again @TwoMoreThings 😄 🎉 

Look forward to see many more contributions from you going forward!

**Comment by nul800sebastiaan** [comment]
Hi there @TwoMoreThings,

First of all: **A big #H5YR** for contributing to Umbraco during Hacktoberfest! We are very thankful for the huge amount of PRs submitted, and all the amazing work you've been doing 🥇 

We have had another record breaking amount of PR's in both the CMS, Documentation, and other smaller public repositories. You can read all about the numbers in the [Umbraco Hacktoberfest round-up blog post](https://umbraco.com/blog/pr-team-update-november-2019-hacktoberfest-round-up/).

Due to the amazing work you and others in the community have been doing, we've had a bit of a hard time keeping up. 😅 While all of the PRs for Hacktoberfest might not have been merged yet, you still qualify for receiving some Umbraco swag, congratulations! 🎉 

In the spirit of Hacktoberfest we've prepared some **exclusive Umbraco swag** for all our contributors - _including you_! 

[Receive your swag!](https://docs.google.com/forms/d/e/1FAIpQLSc-K97Zl5k86cVebKWWqieLM4Fm_jCMGVdbQdYnq4IUJ7EvZg/viewform) :point_left: Please follow this link to fill out and submit the form, **before December 19th 2019**.

Following this date we'll be sending out all the swag, which also means that it might not reach your doorstep before February, so please bear with us and be patient :pray:

The only thing left to say is **thank you so much for participating in Hacktoberfest**! We really appreciate the help!

Kind regards,
The Umbraco PR Team & The Documentation Curators

**Pull request merged**